src/api/services/routing-private-auth-service.ts

```diff
@@ -15,32 +15,47 @@ export class RoutingPrivateAuthService implements CanActivate {
     canActivate(route: ActivatedRouteSnapshot, state: RouterStateSnapshot): Promise<boolean> {
 
         return new Promise(resolve => {
-            if (this.loginService.isLogin) {
-                resolve(this.checkAccess(state.url));
-            } else {
+            if (!this.loginService.isLogin) {
```

Comment:
Also I don't get this... I mean ```if (this.loginService.isLogin || !this.loginService.isLogin)``` we still call ```this.checkAccessLoginUser```.

So it's like saying if user is login or check the access of the user. It doesn't make sense.